packages/slider/demo/stories/SliderStory.tsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { createRef, forwardRef } from 'react';
+import { Story } from '@storybook/react';
+import {
+  ISliderContainerProps,
+  IUseSliderProps,
+  IUseSliderReturnValue,
+  SliderContainer,
+  useSlider
+} from '@zendeskgarden/container-slider';
+import classNames from 'classnames';
+
+interface IComponentProps extends IUseSliderReturnValue {
+  max: IUseSliderProps['max'];
+  min: IUseSliderProps['min'];
+  minThumbRef: IUseSliderProps<Element, HTMLDivElement>['minThumbRef'];
+  maxThumbRef: IUseSliderProps<Element, HTMLDivElement>['minThumbRef'];
+  disabled: IUseSliderProps['disabled'];
+  rtl: IUseSliderProps['rtl'];
+}
+
+const Component = forwardRef<HTMLDivElement, IComponentProps>(
+  (
+    {
+      getTrackProps,
+      getMinThumbProps,
+      getMaxThumbProps,
+      min = 0,
+      max = 100,
```

Comment:
Is there a reason we have to set these values here, even though we're doing this by default in the hook? (Is it for Storybook purposes?)

Reply:
Yes. Garden's standard is to externalize `defaultProps` values in Storybook controls for consumer clarity and to leave the others `undefined`.